python/cugraph/opg/link_analysis/mg_pagerank_wrapper.pyx

```diff
@@ -54,26 +54,37 @@ def mg_pagerank(input_df, local_data, rank, handle, alpha=0.85, max_iter=100, to
 
     cdef uintptr_t c_identifier = df['vertex'].__cuda_array_interface__['data'][0];
     cdef uintptr_t c_pagerank_val = df['pagerank'].__cuda_array_interface__['data'][0];
-
+    
+    cdef uintptr_t c_pers_vtx = <uintptr_t>NULL
+    cdef uintptr_t c_pers_val = <uintptr_t>NULL
+    cdef sz = 0
```

Comment:
I think sz needs a type if I am not mistaken.

Reply:
done